en/docs/reference/connectors/amazonlambda-connector/amazonlambda-connector-example.md

```diff
@@ -234,18 +238,18 @@ You can further refer the application deployed through the CLI tool. See the ins
         "LastUpdateStatusReasonCode": null
    }
    ```
-6. Log in to the AWS Management Console.
+3. Log in to the AWS Management Console.
 
-7. Navigate to the AWS Lambda and Functions tab.
+4. Navigate to the AWS Lambda and Functions tab.
    <img src="{{base_path}}/assets/img/integrate/connectors/awslambdafunction.png" title="Amazon Lambda Function" width="800" alt="Amazon Lambda Function"/>
    
-8. Next you need to execute the function. Navigate to **Configure test events**. <br>
+5. Next you need to execute the function. Navigate to **Configure test events**. <br>
    <img src="{{base_path}}/assets/img/integrate/connectors/configuretestevent.png" title="Configure Test Event" width="800" alt="Configure Test Event"/>
```

Comment:
Please verify if the image is properly aligned. 

 
<img width="1071" alt="Screenshot 2024-07-19 at 2 22 50 PM" src="https://github.com/user-attachments/assets/bc3d7798-0448-4210-b298-da440edb7e0e">

Reply:
This is already aligned
<img width="870" alt="Screenshot 2024-07-22 at 10 55 06" src="https://github.com/user-attachments/assets/78f63f4f-88ba-4610-a80c-3dced66ba1da">